buildenv/jenkins/docker-slaves/s390x/ubuntu16/Dockerfile

```diff
@@ -48,9 +48,6 @@ RUN apt-get update \
     cpio \
     curl \
     file \
-    g++-4.8 \
-    gcc-4.8 \
-    gdb \
```

Comment:
removing gdb intentionally? Why is it not needed?

Reply:
It was not intentional. I'll go and test out the container to see if it is actually needed.

Reply:
This comment is still pending resolution.

Reply:
We tried to test this but were blocked by not having xml parser module in the container. @jdekonin  recalls gdb being added by Colton for testing purposes in order to generate cores and we didn't think it was needed anymore. We could leave it in for now until the parser is added and we can test it properly. Thoughts?

Reply:
If you want to remove it, it can always be quickly added back if a problem is found. Although, according to that explanation we could miss getting a core file for a failure if it is needed. Leaving it in is less likely to break anything.

Reply:
I added gdb back in